### PR TITLE
Docs: add docs for meshery-cilium

### PR DESCRIPTION
**Pull request by iamsdas**
Signed-off-by: Suryashankar Das <suryashankardas.2002@gmail.com>

**Description**
Add initial docs for Meshery adapter for Cilium and reserve the port 10012 for it.
This PR is related to [this](https://github.com/meshery/meshery-cilium/issues/8) issue

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit eec05ce80c7d9c21b4fede8c3fe5d0231afd33ce at: https://61bb91dc6e140e1c4842935c--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 925057b6a29c969710ae33f62976b04f8f915c66 at: https://61bb94819792791ef553736f--meshery-docs-preview.netlify.app

File: docs/_adapters/cilium.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+         1. Prometheus and Grafana connections
+      1. Service Mesh Interface (SMI)
+1. Configuration Analysis, Patterns, and Best Practices
+   1. Custom Service Mesh Configuration
+
+## Lifecycle management
+
+The {{page.name}} can install **{{page.version}}** of {{page.mesh_name}}. A number of sample applications can be installed using the {{page.name}}.
+
+### Install {{ page.mesh_name }}
```
Comment (warunicorn19) [inline docs/_adapters/cilium.md:33]: we don't need the Install for now.

File: docs/_adapters/cilium.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+   1. Using Service Mesh Standards
+      1. Service Mesh Performance (SMP)
+         1. Prometheus and Grafana connections
+      1. Service Mesh Interface (SMI)
+1. Configuration Analysis, Patterns, and Best Practices
+   1. Custom Service Mesh Configuration
+
+## Lifecycle management
+
+The {{page.name}} can install **{{page.version}}** of {{page.mesh_name}}. A number of sample applications can be installed using the {{page.name}}.
```
Comment (warunicorn19) [inline docs/_adapters/cilium.md:31]:
```suggestion
The {{page.name}} can install **{{page.version}}** of {{page.mesh_name}}. A number of sample applications can be installed using the {{page.name}}.

The {{ page.name }} is currently under construction ({{ page.project_status }} state), which means that the adapter is not functional and cannot be interacted with through the <a href="{{ site.baseurl }}/installation#6-you-will-now-be-directed-to-the-meshery-ui"> Meshery UI </a>at the moment. Check back here to see updates.

Want to contribute? Check our [progress]({{page.github_link}}).
```

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

File: docs/_adapters/cilium.md (hunk 1 above)
Comment (warunicorn19) [reply docs/_adapters/cilium.md:33]: You could workload management here with a couple of sample apps like we have on the Tanzu adapter page.

Comment (leecalcote) [reply docs/_adapters/cilium.md:37]: We might want to crack open another issue to track this item.

You're right that we do want to be clear about the state of the adapter.

**Comment by l5io** [comment]
🚀 Preview for commit 4c966953ebf5314bd03c39449db4d3a6abe8e1f1 at: https://61bba44c6b71b330a8c8145e--meshery-docs-preview.netlify.app

**Review by leecalcote — DISMISSED** [review]

File: docs/_adapters/cilium.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Meshery Adapter for Cilium
+name: Meshery Adapter for Cilium
+mesh_name: Ciluim
+earliest_version: v1.10.6
+port: 10012/tcp
+project_status: beta
+adapter_version: v0.0.0
```
Comment (leecalcote) [inline docs/_adapters/cilium.md:9]:
```suggestion
adapter_version: v0.1.0
```

File: docs/_adapters/cilium.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Meshery Adapter for Cilium
+name: Meshery Adapter for Cilium
```
Comment (leecalcote) [inline docs/_adapters/cilium.md:4]:
```suggestion
name: Meshery Adapter for Cilium Service Mesh
```

File: docs/_adapters/cilium.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Meshery Adapter for Cilium
```
Comment (leecalcote) [inline docs/_adapters/cilium.md:3]:
```suggestion
title: Meshery Adapter for Cilium Service Mesh
```

**Pull request merged**